RELEASES.md

```diff
@@ -0,0 +1,114 @@
+
+1.  They should be well-tested by the community before we release.
+2.  They often have architectural implications for the entire Storybook ecosystem, so should be discussed thoroughly before release. Doing `alpha` releases allow us to test in the community without necessarily achieving agreement. For example, currently `Vue` support is in `alpha`, although it contains potentially controversial changes to addons.
+3.  They often deserve proper marketing treatment (blog posts, release announcement, podcast, etc.)
+
+Therefore we merge these into a release branch of the form `release/MAJOR.MINOR`
+and we create preview releases and get them tested by the community before
+merging those branches into `master`.
+
+> NOTE: it is possible to edit the `base` branch in a PR, so we can expect users to issue PRs to `master`, but as maintainers as we see feature releases we should set them to merge into the appropriate release branch.
+
+In general we should release 2-3 headline features per minor release for
+marketing purposes. Each headline feature should get its own blog post on the
+Storybook medium publication, and the release itself should also get a blog
+post.
+
+## MAJOR releases
+
+We should avoid major releases, and should try to do these at most 1-2x per
+year. Ideally each breaking change would have been agreed upon by the
+maintainers and validated in backwards-compatible feature releases. At some
+point once a change has been thoroughly vetted, we deprecate the old way and
+give some time for people to upgrade to the new way. Finally MAJOR releases
+should have killer features that reward users for upgrading, and should not be
+used to scratch our own architectural itches -- unless those itches are really
+killing development in some meaningful way.
+
+## Blocking bugs
+
+Most PATCH releases come from community members, who generously fix problems as
+they come up. However, there are also bugs that never get picked up and just sit
+there gathering upvotes and "me too" comments. We need a way to make sure that
+these bugs get addressed.
+
+For every non-PATCH release, we nominate a small number of bugs that must be
+addressed before a release can go out by adding them to the milestone. For example, here's a list of blocking bugs [for the 3.2 milestone](https://github.com/storybooks/storybook/milestone/3).
+
+Adding bugs to the milestone helps people looking for good ways to contribute,
+or to understand what is blocking the release so they can actually do something
+about it. Discussion about which bugs are critical happens in the `#maintenance`
+channel [in our Slack](https://storybooks-slackin.herokuapp.com/).
+
+If you're experiencing a bug, the best way to make sure that it gets attention
+is to upvote it by adding a "thumbs-up" reaction in Github. This way important
+bugs quickly bubble to the top [with a
+search](https://github.com/storybooks/storybook/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).
```

Comment:
Can we change this link to filter out non-`bug`s, `fixed` bugs, bugs with a milestone, etc